node/src/chain_spec.rs

```diff
@@ -363,6 +363,9 @@ fn parachain_genesis(
 			members: tech_committee_members,
 			phantom: Default::default(),
 		},
+		orml_nft: OrmlNftConfig {
```

Comment:
Ok we should set a guideline about naming abbreviations. I can see we're doing Id.